docs/design.md

```diff
@@ -119,3 +119,9 @@ and most of the validator teams have trouble managing logs from running nodes du
 ![monitoring_to_be](https://user-images.githubusercontent.com/6308023/210969235-4aa505ee-28cc-4e16-8129-843dbc4f2ca0.png)
 
 **VATZ** will support to more monitoring and analysis 3rd party apps tool as shown in the diagram above.
+
+---
+
+# VATZ Project Design (v2)
+TBD, Will be release on 2024.
```

Comment:
The detailed design for VATZ v2 is currently under development and will be released in 2024. If you have any questions or comments regarding VATZ v2, please feel free to contact us.